src/main/java/net/snowflake/client/jdbc/SnowflakeConnectionV1.java

```diff
@@ -63,7 +58,7 @@
 
   private static final String SSL_NATIVE_PROTOCOL = "https";
 
-  private boolean isClosed = true;
+  private boolean isClosed;
```

Comment:
isClosed to be false when an instance is created. It makes sense because the connection is made when the instance is created.